src/main/lua/exasol/adapter/databricks/authentication.lua

```diff
@@ -8,24 +8,34 @@ local M = {}
 
 ---@alias TokenProvider fun(): string
 
+---@param user string?
+---@param password string?
+---@return string header_value
+local function basic_auth_header(user, password)
+    return "Basic " .. base64.encode(string.format("%s:%s", user, password))
+end
+
+---Fetch Databricks OAuth token.
+---See https://docs.databricks.com/en/dev-tools/auth/oauth-m2m.html#manually-generate-and-use-access-tokens-for-oauth-m2m-authentication
 ---@param connection_details DatabricksConnectionDetails
 ---@return string
 local function fetch_oauth_token(connection_details)
-    log.trace("Fetching new OAuth M2M token")
+    local url = connection_details.url .. "/oidc/v1/token"
```

Comment:
Do we create a new access token for each Virtual Schema connection?
Are there any limitations from Databricks with regards to number of these tokens?

Reply:
Tokens are valid for 1h. We create a new token for every request. There is no limitation for the number of tokens.